lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -64,6 +65,7 @@ const initializeGranulesAndFilesMigrationResult = (): GranulesAndFilesMigrationR
  * @param {Object} record
  *   Record from DynamoDB
  * @param {Knex.Transaction} trx - Knex transaction
+ * @param {Object} migrationParams
```

Comment:
Could we add a very brief explanation about the two valid options here in the docstring?